server/xpub-controller/manuscript.js

```diff
@@ -18,9 +18,12 @@ class Manuscript {
       )
     }
 
-    const originalEmails = manuscript.getAuthor()
+    const originalEmails = (manuscript.getAuthor()
       ? manuscript.getAuthor()
-      : [].map(author => author.alias.email).join(',')
+      : []
+    )
```

Comment:
```suggestion
  const originalEmails =  manuscript.getAuthor() || []
```

Reply:
Sorry that was prettify that refactored all that :/